src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -120,7 +120,8 @@ Replayer<I>::Replayer(
     m_state_builder(state_builder),
     m_replayer_listener(replayer_listener),
     m_lock(ceph::make_mutex(librbd::util::unique_lock_name(
-      "rbd::mirror::image_replayer::snapshot::Replayer", this))) {
+      "rbd::mirror::image_replayer::snapshot::Replayer", this))),
+    m_update_watch_ctx(nullptr) {
```

Comment:
I'd initialize it with an in-class initializer in Replayer.h, it's safer that way.